medusa/backup.py

```diff
@@ -57,10 +57,35 @@ def generate_md5_hash(src, block_size=BLOCK_SIZE_BYTES):
     return base64_md5
 
 
+def md5_multipart(src, part_size):
```

Comment:
The `part_size` seems to be more of a `part_count`. But as I'm saying in other comments, I'd suggest dropping it.

Reply:
it's actually a size, but I agree on dropping it to rely on a constant instead.